chart/values.yaml

```diff
@@ -303,6 +303,21 @@ workers:
       limits:
         cpu: 0
     tolerations: []
+  -
+    # name of the deployment
+    deployName: "light"
+    # Maximum number of jobs running at the same time for the same namespace
+    maxJobsPerNamespace: 1
+    # job types that this worker can process
+    workerOnlyJobTypes: "dataset-split-names-from-streaming,dataset-split-names-from-dataset-info"
```

Comment:
this worker could contain all the "light" jobs (parquet, dataset-info, sizes, etc)

Reply:
Sure, Done in https://github.com/huggingface/datasets-server/pull/945/commits/47e16e3cdb88b1005c34bec34c181162008fe1d3. I also find this more secure so that we dont loose any new Job Runner by mistake.